setup-jest.ts

```diff
@@ -1,9 +1,16 @@
 import { enableFetchMocks } from 'jest-fetch-mock';
 import { Crypto } from '@peculiar/webcrypto';
+import { WorkOS } from './src/workos';
+import { WebIronSessionProvider } from './src/common/iron-session/web-iron-session-provider';
 
 enableFetchMocks();
 
 // Assign Node's Crypto to global.crypto if it is not already present
 if (!global.crypto) {
-    global.crypto = new Crypto();
+  global.crypto = new Crypto();
 }
+
+// For tests, we can use the WebIronSessionProvider
+WorkOS.prototype.createIronSessionProvider = jest
+  .fn()
+  .mockReturnValue(new WebIronSessionProvider());
```

Comment:
Don't love this testing edit. Alternatively I could pass the iron-session-provider in the constructor WorkOS, but feels a bit odd given we only use it for User Management.

Reply:
I think it's fine, it would be weird if we required it when you're only interested in SSO or one of the other products.